### PR TITLE
Various status improvements

### PR DESCRIPTION
**Pull request by whalemer**

### CODE REVIEW

File: server/chat-commands.js [hunk 1]
```diff
@@ -716,6 +714,23 @@ const commands = {
 	unaway: 'back',
 	unafk: 'back',
 	back(target, room, user) {
+		if (target) {
+			// Clearing another user's status
```
Comment (Zarel) [inline server/chat-commands.js]: This should probably be a different command?

Comment (Zarel) [reply server/chat-commands.js]: Especially since it has exactly nothing in common with the regular command.

File: server/chat-commands.js [hunk 2]
```diff
@@ -716,6 +714,23 @@ const commands = {
...
+			let reason = this.splitTarget(target);
+			let targetUser = this.targetUser;
+			if (!targetUser) return this.errorReply(`User '${target}' not found.`);
+			if (!targetUser.status) return this.errorReply(`${targetUser.name} does not have a status set.`);
+			if (!this.can('forcerename', targetUser)) return false;
+
+			let bracketIndex = targetUser.status.indexOf(')');
+			if (bracketIndex < 0) bracketIndex = -2; // Someone's trying to clear the "Idle" status, for some reason
```
Comment (Zarel) [inline server/chat-commands.js]: Why is it `Idle` and not `(Idle)`, anyway?

Comment (scheibo) [reply server/chat-commands.js]: Check commits, its been fixed to be consistent

**Pull request merged**